capstone/src/main/webapp/script.js

```diff
@@ -390,17 +421,12 @@ function configModal() {
   // Get the modal
   const modal = document.getElementById('modal');
 
-  // Get the button that opens the modal
-  const btn = document.getElementById('associations-container');
-
   // Get the <span> element that closes the modal
   const span = document.getElementById('modal-close');
 
   // When the user clicks the button, open the modal
```

Comment:
This comment probably should be removed with the current use of configModal. In addition, we might want to change the name of the method to something like openModal with its current usage.

Reply:
I don't know which comment you are referring to? The get the model, get the <span>, and open the model comments are all yours and I think that they are fine because they correctly describe what the lines below them do. configModal also isn't a a bad name since there is a comment that explains it opens the model also when the user clicks a button.

Reply:
At first I was a little bit confused as to why the modal.style.display = 'block' and loadWordcloud(); lines weren't in an onclick function anymore. I think modifying the comment above these lines, changing the name of configModal, or adding a comment above the method would help make this more clear, especially since the file is so large.

Reply:
Added a comment as per your request.